sponsorship_compassion/models/contracts.py

```diff
@@ -532,9 +546,23 @@ def contract_active(self):
                 lambda c: 'S' in c.type and not c.is_active):
             # UpsertConstituent Message
             partner = contract.correspondant_id
-            partner.upser_constituent()
-
-            # TODO : Create Sponsorship Message
+            partner.upsert_constituent()
+
+            # Create Sponsorship Message
+            message_obj = self.env['gmc.message.pool']
+            action_id = self.env.ref(
+                'sponsorship_compassion.create_sponsorship').id
+
+            messages = message_obj.search([
+                ('partner_id', '=', partner.id),
+                ('state', '=', 'new'),

```

Comment:
Change with `('state', 'in', ('new', 'pending'))`. Can you as well change it in the method `upsert_constituent` ?